urbanairship-accengage-react-native/package.json

```diff
@@ -40,7 +40,7 @@
   "scripts": {
     "prepare": "bob build",
     "test": "jest --verbose",
-    "generate-docs": "typedoc --out docs --exclude js/__tests__ js/"
+    "generate-docs": "typedoc --out docs"
```

Comment:
The exclude of `js/__tests__` and the include of `js/` are covered by the `tsconfig.json` file, so they don't need to be here.